component/file-storage/src/main/java/org/exoplatform/commons/file/services/util/Utils.java

```diff
@@ -0,0 +1,17 @@
+package org.exoplatform.commons.file.services.util;
```

Comment:
Is it mandatory to have utils class to call listenerService ?
I think you should call it directly, without creating utils class

Reply:
Well, i just wanted to not duplicate try/catch bloc

Reply:
I prefer to duplicate the try catch block, that allow us to make different error log messages, instead of adding a new Utils class. I find it more readable.

Reply:
Ok